lib/cc/engine/analyzers/scala/main.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "flay"
+require "json"
+require "cc/engine/analyzers/reporter"
+require "cc/engine/analyzers/analyzer_base"
+
+module CC
+  module Engine
+    module Analyzers
+      module Scala
+        class Main < CC::Engine::Analyzers::Base
+          LANGUAGE = "scala".freeze
+          PATTERNS = ["**/*.sc", "**/*.scala"].freeze
+          DEFAULT_MASS_THRESHOLD = 60
+          DEFAULT_FILTERS = [].freeze
```

Comment:
Do you think we need to ignore imports by default? That's a common source of false positives in many languages.

Reply:
I think I haven't hit issues with the current `DEFAULT_MASS_THRESHOLD`. Improving the spec so I can adjust this! 👍